perfkitbenchmarker/benchmarks/hadoop_terasort_benchmark.py

```diff
@@ -121,29 +72,13 @@ def Prepare(benchmark_spec):
         required to run the benchmark.
   """
   vms = benchmark_spec.vms
-
   master = vms[0]
-  slaves = vms[1:]
-
-  master_ip = master.internal_ip
-  slave_ips = [vm.internal_ip for vm in slaves]
+  workers = vms[1:]
 
-  args = [((vm, master_ip, slave_ips), {}) for vm in vms]
-  vm_util.RunThreaded(InstallHadoop, args)
-
-  conf_dir = _ConfDir(master)
-  master.RemoteCommand('yes | hadoop-%s/bin/hdfs namenode -format' %
-                       HADOOP_VERSION)
-  master.RemoteCommand(('hadoop-%s/sbin/hadoop-daemon.sh --config %s '
-                        '--script hdfs start namenode') % (HADOOP_VERSION,
-                                                           conf_dir))
-  master.RemoteCommand(('hadoop-%s/sbin/yarn-daemon.sh --config %s '
-                        'start resourcemanager') % (HADOOP_VERSION, conf_dir))
-
-  master.RemoteCommand(('hadoop-%s/sbin/mr-jobhistory-daemon.sh --config %s '
```

Comment:
Looks like the new setup doesn't start the jobhistory server - I seem to remember this being necessary (at least on Azure - see https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/pull/151). Have you run on AWS and Azure? If it works on them without the jobhistory server then that works for me.